docs/pages/workflow/debugging.md

```diff
@@ -55,41 +56,65 @@ This might indicate that there is a performance issue. You likely need to run yo
+- Android Emulator: Either press <kbd>Cmd ⌘</kbd> + <kbd>M</kbd> or <kbd>Ctrl</kbd> + <kbd>M</kbd> or run `adb shell input keyevent 82` in your terminal
 
-The Developer Menu gives you a couple different functionalities. A few are pretty self-explanatory, like:
+The Developer Menu looks like this:
 
-- Reload manifest & JS bundle: this will reload your app. Usually this isn't necessary if you have Live or Hot Reload enabled, since it will automatically refresh whenever you save your changes in your text editor.
-- Go to Expo Home: Leave your app and navigate back to the Expo Go app homescreen
-- Enable/Disable Live Reload: When enabled, your app will automatically refresh the JS bundle whenever you save file changes in your project directory.
+<ImageSpotlight alt="The Expo Go Developer Menu, showing the menu options available" src="/static/images/expo-go-developer-menu.png" containerStyle={{ paddingBottom: 0 }} />
+
+The Developer Menu provides multiple options. A few are pretty self-explanatory, like:
+
+- Reload: reload your app - usually not necessary if you have Fast Refresh enabled
+- Copy link to clipboard: Copy the [`exp://` link to your app](../guides/linking.md)
+- Go to Home: Leave your app and navigate back to the Expo Go app homescreen
+- Enable/Disable Fast Refresh: Toggle automatic refreshing the JS bundle whenever you make changes to files in your project
 
 Now let's explore some of the more exciting functionalities...
 
+#### Debug Remote JS
+
+Opens a React Native Debugger tab in your browser, thanks to [`@react-native-community/cli-debugger-ui`](https://github.com/react-native-community/cli/tree/main/packages/cli-debugger-ui):
+
+<ImageSpotlight alt="The Expo Go Element Inspector, showing details about an element after inspecting it" src="/static/images/react-native-community-cli-debugger-ui.png" containerStyle={{ paddingBottom: 0 }} />
+
+This allows you to use your browser's DevTools for the Console tab (for using `console.log`).
+
+⚠️ The Network tab will not work out of the box! To enable the Network tab and other debugging tools, additional setup is required - see the [React Native Debugger](#react-native-debugger) and [React DevTools](#debugging-with-react-devtools) sections below.
+
 #### Show Performance Monitor
 
-This opens up a small window giving you performance information of your app. You will see:
+Opens up a small window giving you performance information of your app. You will see:
 
 - RAM usage of your project
 - JavaScript heap (this is an easy way to know of any memory leaks in your application)
 - 2 numbers for Views, the top indicates the number of views for the screen, the bottom indicates the number of views in the component
 - Frames Per Second for the UI and JS threads. The UI thread is used for native Android or iOS UI rendering. The JS thread is where most of your logic will be run, including API calls, touch events, etc.
```

Comment:
```suggestion
- Frames Per Second for the UI and JS threads. The UI thread is used for native Android or iOS UI rendering. The JS thread is where most of your logic runs, including API calls, touch events, and so on.
```

Unrelated to the changes in this PR but I think we can remove the latin abbreviation as per the writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations